packages/quantic/cypress/e2e/default-1/sort/sort.cypress.ts

```diff
@@ -33,14 +37,60 @@ describe('quantic-sort', () => {
   ];
   const sortOptionValues = sortOptions.map((option) => option.value);
 
+  const customSortOptions = [
+    {
+      label: 'Date ascending',
+      value: 'date ascending',
+      criterion: {
+        by: 'date',
+        order: 'ascending',
+      },
+    },
+    {
+      label: 'Views Descending',
+      value: '@ytviewcount descending',
+      criterion: {
+        by: 'field',
+        field: 'ytviewcount',
+        order: 'descending',
+      },
+    },
+    {
+      label: 'No Sort',
+      value: 'nosort',
+    },
+  ];
+
+  const defaultCustomOptionValue = 'date ascending';
+
+  const customSortOptionsValues = customSortOptions.map(
+    (option) => option.value
+  );
   interface SortOptions {
     useCase: string;
+    withCustomSortOptions?: boolean;
+    withInvalidCustomSortOptions?: boolean;
   }
 
-  function visitSort(options: Partial<SortOptions>, waitForSearch = true) {
+  interface SortActionsOptions {
+    withCustomSortOptions?: boolean;
+    withInvalidCustomSortOptions?: boolean;
+  }
+
+  function visitSort(
+    options: Partial<SortOptions>,
+    waitForSearch = true,
+    actionsOptions: Partial<SortActionsOptions>
+  ) {
```

Comment:
Since almost every time `waitForSearch` is set to true when calling `visitSort`, it's better to switch the order of the parameters and also give a default value for the `actionsOptions` in order to just be able to call `visitSort({...})` with only one parameter in the default cases.
```suggestion
  function visitSort(
    options: Partial<SortOptions>,
    actionsOptions: Partial<SortActionsOptions> = {}
    waitForSearch = true,
  ) {
```